boards/silabs/dev_kits/sltb004a/Kconfig.defconfig

```diff
@@ -9,4 +9,7 @@ config CMU_HFXO_FREQ
 config CMU_LFXO_FREQ
 	default 32768
 
+config BOARD_LATE_INIT_HOOK
+	def_bool y if CCS811
```

Comment:
Since `BOARD_LATE_INIT_HOOK` is defined elsewhere, I believe `def_bool` is not relevant and should be replaced by `default`, no?